packages/legend-graph/src/graph-manager/action/query/Query.ts

```diff
@@ -37,8 +33,9 @@ export class QueryParameterValue {
 export abstract class QueryExecutionContext {}
 
 export class QueryExplicitExecutionContext extends QueryExecutionContext {
-  mapping!: PackageableElementReference<Mapping>;
-  runtime!: PackageableElementReference<PackageableRuntime>;
+  // NOTE: Query can be built before the actual graph is built so we can't have the reference of metamodels here
+  mapping!: string;
```

Comment:
setUpEdtiorState() has been updated to fetch query instead of LightQuery anymore
https://github.com/finos/legend-studio/pull/3488/files#diff-5dae4710ada9d402191df7256d5cc7d4502e3b40a8f1b9acf960143f7ee6adfeR1293

Therefore, it needs to be changed to use string